### PR TITLE
feat: Add ignorePath option in cli (infuse, squeeze) v2 version

### PR DESCRIPTION
**Pull request by kiibo382**
fix: #149 

add ignorePath to yargs.
add 'ignore' (npm package), and make sure to load .ignore file.

### CODE REVIEW

**Comment by kiibo382** [comment]
This branch is Vue2.x version and from maintain-v1 branch.

File: test/fixtures/.ignore-i18n [hunk 1]
```diff
@@ -0,0 +1 @@
+**/hello.vue
```
Comment (kazupon) [inline test/fixtures/.ignore-i18n:1]: Could you test with the fixture file path? (for example, `src/pages/Login.vue`)
`**/hello.vue` is not the fixture file path.
In unit tests, We need to use the fixture file path.

**Review by kazupon — CHANGES_REQUESTED** [review]

File: test/fixtures/.ignore-i18n (hunk 1 above)
Comment (kiibo382) [reply test/fixtures/.ignore-i18n]: Thank you for your review!
Please check my commit!

**Review by kazupon — APPROVED** [review]

**Comment by kazupon** [comment]
@kiibo382 
Could you fix the CI failure?

**Comment by kiibo382** [comment]
> @kiibo382
> Could you fix the CI failure?

Oh, sorry.
I forgot "npm run lint".
I fix it, please rerun.

**Pull request merged**